bin/build-service-images

```diff
@@ -47,12 +47,13 @@ function build_all {
   parallel=$1
   pids=()
   for dir in services/*; do
+    [[ $dir == "services/README.md" ]] && continue
```

Comment:
Might be better to use `if [ -d "$dir" ]` here to only process `dir` if it actually is a directory. That way if we add another file besides README.md at some point things won't break.